src/com/lftechnology/java/training/alina/collections/generics/instantiateproduct/Product.java

```diff
@@ -0,0 +1,49 @@
+
+package com.lftechnology.java.training.alina.collections.generics.instantiateproduct;
+
+/**
+ * Product Class consists of products of different data types
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ * @param <T1>
+ * @param <T2>
+ */
+public class Product<TA, TB> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this generic name to match the regular expression '^[A-Z]$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00119)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this generic name to match the regular expression '^[A-Z]$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00119)